### PR TITLE
update memory list

### PR DESCRIPTION
**Pull request by AnassM06**
Changes:
 search memories by description or title

How to test: 
 save memory and search on title description

Screenshots:
If applicable, add a screenshot or video here to show the new feature

### CODE REVIEW

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java [hunk 1]
```diff
@@ -1,26 +1,86 @@
...
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable s) {
+                filter(s.toString());
+
+
+            }
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java:59]: Ik had een `SimpleTextWatcher` gemaakt. 
Dan kan je dit:
```java
 input_search.addTextChangedListener(new TextWatcher() {
            @Override
            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
             }
             @Override
            public void onTextChanged(CharSequence s, int start, int before, int count) {
             }
             @Override
            public void afterTextChanged(Editable s) {
                filter(s.toString());
             }
}
```
vervangen met dit:
```java
 input_search.addTextChangedListener(new SimpleTextWatcher() { 
           @Override
            public void afterTextChanged(Editable s) {
                filter(s.toString());
             }
}
```

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Nice work! I have one small improvement in code though.

**Review by Almightywesley — COMMENTED** [review]
The sort function does not work propperly i give the comments irl

**Comment by MrTheGood** [comment]
@Almightywesley sorting isn'nt part of this PR.

**Review by MrTheGood — APPROVED** [review]

**Review by Almightywesley — APPROVED** [review]
Anas you have implemented the feefback that i gave you nice job! it works

**Pull request merged**